data/json/recipes/chem/mutagen.json

```
@@ -795,5 +795,19 @@
     "using": [ [ "serum_production_standard", 37 ] ],
     "components": [ [ [ "mutagen_rabbit", 2 ] ] ],
     "flags": [ "SECRET" ]
+  },
+  {
+    "type": "recipe",
+    "result": "purifier_smart_shot",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_MUTAGEN",
+    "skill_used": "cooking",
+    "skills_required": [ "firstaid", 6 ],
+    "difficulty": 10,
+    "time": 120000,
```

Just realized it uses old time units.
```suggestion
    "time": "2 h",
```